### DISCUSSION TITLE
Post request node in flowise

### DISCUSSION THREAD

**Discussion by alexsstockton** [FlowiseAI/Flowise#2980] [Q&A]
I'm struggling to use the post request node in the flowise editor. It looks like the tool is trying to send the llm output but I'm receiving nothing at the endpoint. How do I transfer the output of an llm or agent node into the post request node?

Appreciate y'all!

**Comment by NatEvory** [comment]
I am having the same issue. I have tried many different descriptions, but I never get any data in the post body.
The tool is being called, and I'm seeing the request, it's just not sending data.
